lesson8-UnitTesting/FizzBuzz/FizzBuzzConsole/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Linq;
+
+namespace FizzBuzzConsole
```

Comment:
General remark, it's a good approach to name namespaces according to format `[Company(if applicable)].[Feature].[Component].[Subcomponent]`. See also https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-namespaces.